tests/unittests/tests-core/tests-core-clist.c

```diff
@@ -16,7 +16,8 @@
 
 #define TEST_CLIST_LEN    (8)
 
-static clist_node_t tests_clist_buf[TEST_CLIST_LEN];
+static list_node_t tests_clist_buf[TEST_CLIST_LEN];
```

Comment:
Here and below: the datatype is called `list` now so the variable, function and file names should reflect that.